### ISSUE TITLE
Firefox no longer respects config in ~/.mozilla/ ?

### ISSUE THREAD

**Issue by takeda** [nix-community/home-manager#1216]
I'm trying to use home-manager to configure Firefox, but it doesn't look like it has any effect. Neither of extensions that I provide works. I read that Firefox recently disabled side loading, so I suspect it is related to that, but commit messages say that it should still works, only now there more likely will be conflicts. It also looks like Firefox is storing configuration in ~/.cache/ now. This is on NixOS 20.03.

**Comment by teto** [comment]
I tried configuring  firefox via HM recently too. I managed to generate a similar /home/teto/.mozilla/firefox/profiles.ini apart from `Version=2` but now firefox -ProfileManager fails to launch one of the profile with `JavaScript error: resource://gre/modules/XULStore.jsm, line 66: Error: Can't find profile directory`. the strace seems to look for `~/home-manager/<profile>`. I don't have any previous experience nor I know enough about firefox to check why. It seems like one can't set absolute profile paths . `firefox -P <profile_name> `still works though
